crates/weaver_codegen_test/semconv_registry/registry/system.yaml

```diff
@@ -16,6 +16,7 @@ groups:
               value: 'nice'
             - id: idle
               value: 'idle'
+              stability: stable
```

Comment:
Note. For our internal codegen test to pass I now had to explicitly set this to stable!